app/experimenter/templates/experiments/list.html

```diff
@@ -145,12 +149,7 @@ <h5>{{ experiment.population }}</h5>
     </div>
 
     {% for field in filter.form %}
-      {% if field.name == "archived" %}
-        <label>
-          {{ field }}
-          {{ field.label }}
-        </label>
-      {% elif field.name == "in_qa" %}
+      {% if field.name == "in_qa" or field.name == "surveys" or field.name == "archived" %}
```

Comment:
I also realized that we had some redundant pieces of template here that could be combined into one larger `if` statement.